CMakeLists.txt

```diff
@@ -206,7 +206,7 @@ endif()
 unset(check_pie_output)
 
 # The core_interface library aims to encapsulate common build flags.
-# It is intended to be a usage requirement for all other targets.
+# It is intended to be a usage requirement for all C++ targets.
```

Comment:
My question was about "is intended to be". If there are exceptions to this usage, would be good to document them. Either way, C++ targets isn't technically correct right? This will also be applied to atleast ObjC++ code on macOS.